### ISSUE TITLE
Alerts on Environments page don't receive focus on appearing

### ISSUE THREAD

**Issue by manfromjupyter** [nbgallery/nbgallery#794]
On environment page (and most likely others), for low vision users (and all users for that matter), if they have 10 environments, scroll to bottom, and delete one (but also probably edit or create one), the success message is not seen.

Probable solution is a JS "Scroll up" for all alerts.

**Comment by mcrutch** [comment]
makeAlert ATTEMPTS to scroll into view already

**Comment by manfromjupyter** [comment]
Agreed. It's just not working. Even when I manually trigger a scrollup it's not working. There's something super weird happening I have been unable to really figure out

**Comment by manfromjupyter** [comment]
@manfromjupyter Look into this after #948. Small chance it's fixed by removing 2 of those 7 endpoints and moving all of the JS into one place
